examples/with-tailwind/apps/web/src/pages/index.tsx

```diff
@@ -10,15 +28,21 @@ export default function Home() {
       </Head>
 
       <main className="mx-auto w-auto px-4 pt-16 pb-8 sm:pt-24 lg:px-8">
-        <h1 className="mx-auto max-w-5xl text-center text-6xl font-extrabold leading-[1.1] tracking-tighter text-white sm:text-7xl lg:text-8xl xl:text-8xl">
-          Web <br className="hidden lg:block" />
-          <span className="inline-block bg-gradient-to-r from-brandred to-brandblue bg-clip-text text-transparent">
+        <h1 className="mx-auto text-center text-6xl font-extrabold tracking-tight text-white sm:text-7xl lg:text-8xl xl:text-8xl">
+          Web
+          <div className="bg-gradient-to-r from-brandred to-brandblue bg-clip-text text-transparent">
```

Comment:
divs aren't allowed as children to heading tags. any reason for this change?

you can validate with this markup here: https://validator.w3.org/nu/#textarea

```html
<!DOCTYPE html>
<html lang=en>
<head><title>hi</title></head>
<h1>
<div>something</div>
</h1>
</html>
```